tests/Repository.py

```diff
@@ -481,13 +481,6 @@ def testCreateRepositoryDispatch(self):
         without_payload = self.repo.create_repository_dispatch("type")
         self.assertTrue(without_payload)
 
-    @mock.patch("github.PublicKey.encrypt")
```

Comment:
Same here, you can delete the corresponding `ReplayData` file.

Reply:
removed from repo